helpers.c

```diff
@@ -313,4 +313,166 @@ hsl_to_rgb_float (float *h_, float *s_, float *l_)
   *l_ = b;
 }
 
+// RGB to RYB
+// adapted from http://www.deathbysoftware.com/colors/index.html
+	/**********************************************************************************************
+	 * Given a RYB color, calculate the RGB color.  This code was taken from:
+	 * 
+	 * @param iRed     The current red value.
+	 * @param iYellow  The current yellow value.
+	 * @param iBlue    The current blue value.
+	 * 
+	 * http://www.insanit.net/tag/rgb-to-ryb/
+	 * 
+	 * Author: Arah J. Leonard
+	 * Copyright 01AUG09
+	 * Distributed under the LGPL - http://www.gnu.org/copyleft/lesser.html
+	 * ALSO distributed under the The MIT License from the Open Source Initiative (OSI) - 
+	 * http://www.opensource.org/licenses/mit-license.php
+	 * You may use EITHER of these licenses to work with / distribute this source code.
```

Comment:
Arah J. Leonard also indicated it is LGPL, which seems to also be compatible with ISC according to wikipedia.  I literally copied and pasted from rgb_ryb.js from http://www.deathbysoftware.com/colors/ryb.zip and it had all that license verbage above the functions.  Do we need to include the license from Joshua Gentry even though I only took code from Arah?  I suppose Joshua converted it from Python to Javascript. . .   Here's Joshua's license:

/**************************************************************************************************
 *  Color Space - A RGB to RYB converter.
 *  
 *  Copyright (C) 2013 Joshua Gentry
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU Affero General Public License as
 *  published by the Free Software Foundation, either version 3 of the
 *  License, or (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU Affero General Public License for more details.
 *
 *  You should have received a copy of the GNU Affero General Public License
 *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */